go/host/enclave/guardian.go

```diff
@@ -608,3 +646,40 @@ func (g *Guardian) streamEnclaveData() {
 		}
 	}
 }
+
+func (g *Guardian) getBatchesAfterSize(seqNo uint64) (uint64, error) {
```

Comment:
Find this method name a bit confusing. Maybe could be something like `getSizeOfBatchesAfter()` (maybe `since` instead of `after`)

Reply:
Yeah, I found that confusing as well

Reply:
Third here, hate the name. WDYT of`calculateUnrolledupBatchesSize` ?